packages/auth/extension/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@firebase/auth/extension",
```

Comment:
Ack. Renamed 'extension' to 'chrome-extension' here and other places (including folders' names). Thanks!

Reply:
`web-extension` would be the preferred nomenclature. Since I presume we would want these to work in firefox and safari

Reply:
While Firebase Auth only supports Chrome extensions currently, I changed it to 'web-extension' in case we support other web extensions in the future. Thanks!

Reply:
absolutely understand, appreciate the update